File: Kapsch.Core.Gateway.Models/Configuration/AddressInfoModel.cs
using System;

namespace Kapsch.Core.Gateway.Models.Configuration
{
    public class AddressInfoModel
    {
        public long ID { get; set; }

        public long AddressTypeID { get; set; }

        public long SourceID { get; set; }

        public long PersonInfoID { get; set; }

        public string Line1 { get; set; }

        public string Line2 { get; set; }

        public string Suburb { get; set; }

        public string Town { get; set; }

        public string Country { get; set; }

        public string Code { get; set; }

        public decimal? Latitude { get; set; }

        public decimal? Longitude { get; set; }

        public long? CreatedUserDetailID { get; set; }

        public DateTime CreatedDate { get; set; }

        public long? IsPrefferedIndicator { get; set; }
    }
}

File: Kapsch.MobileApps/KapschMobile/1_iTicket/src/main/java/za/co/kapsch/iticket/Models/AddressInfoModel.java
package za.co.kapsch.iticket.Models;

import android.os.Parcel;
import android.os.Parcelable;

import com.google.gson.annotations.SerializedName;

import java.util.Date;

/**
 * Created by csenekal on 2016-08-02.
 */
public class AddressInfoModel{

    @SerializedName("ID")
    private long mID;

    @SerializedName("AddressTypeID")
    private long mAddressTypeID;

    @SerializedName("SourceID")
    private long mSourceID;

    @SerializedName("PersonInfoID")
    private long mPersonInfoID;

    @SerializedName("Line1")
    private String mLine1;

    @SerializedName("Line2")
    private String mLine2;

    @SerializedName("Suburb")
    private String mSuburb;

    @SerializedName("Town")
    private String mTown;

    @SerializedName("Country")
    private String mCountry;

    @SerializedName("Code")
    private String mCode;

    @SerializedName("Latitude")
    private Double mLatitude;

    @SerializedName("Longitude")
    private Double mLongitude;

    @SerializedName("CreatedUserDetailID")
    private Long mCreatedUserDetailID;

    @SerializedName("CreatedDate")
    private Date mCreatedDate;

    @SerializedName("IsPrefferedIndicator")
    private Long mIsPrefferedIndicator;
}
